backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -326,6 +335,113 @@ public void delete(BacktraceDatabaseRecord record) {
         this.backtraceDatabaseContext.delete(record);
     }
 
+    @Override
+    public boolean enableBreadcrumbs(Context context) {
```

Comment:
do we need to pass once again a context if we already stored context in Backtrace client for example?

Reply:
Outdated, now we will pass the context from `BacktraceBase` directly to `Breadcrumbs`